package/plugin-gargoyle-qr-code/files/www/hooks/login/090-qr_code.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * This program is copyright © 2020 Rouven Spreckels and is distributed under the terms of the GNU GPL
+ * version 2.0 with a special clarification/exception that permits adapting the program to
+ * configure proprietary "back end" software provided that all modifications to the web interface
+ * itself remain covered by the GPL.
+ * See http://gargoyle-router.com/faq.html#qfoss for more information
+ */
+
+// ECMAScript 6 (ES6) Strict Mode Code
+"use strict";
+
+addLoadFunction(() =>
```

Comment:
"Arrow Functions" irk me a little. I won't say no, but please carefully consider their use.
This is probably an old fashioned comment. 😄

Reply:
I see that, especially here with a multi-line function in almost-global scope. Will change it in these places. But I would suggest to use arrow functions at least in places with a lot of one-liners like adding a set of event listeners or with array combinators:

`map(id => byId(id))` compared to `map(function(id) { return byId(id); })`.